src/mixins/nft.js

```diff
@@ -293,17 +294,23 @@ export default {
     },
 
     nftEditions() {
-      // TODO: Use real stock from API
-      return [
-        {
-          name: 'Standard Edition',
-          priceLabel: this.formattedNFTPriceInLIKE,
-          value: 'standard',
-          stock: this.nftIsCollectable ? 500 : 0,
-        },
-      ];
+      const prices = this.getNFTBookStorePricesByClassId(this.classId);
+      return prices
+        ? prices.map((edition, index) => ({
+            name: edition.name,
+            priceLabel: formatNumberWithUnit(edition.price, 'USD'),
+            value: index,
+            stock: edition.stock,
+          }))
+        : [
+            {
+              name: 'Standard Edition',
+              priceLabel: this.formattedNFTPriceInLIKE,
+              value: 'standard',
+              stock: this.nftIsCollectable ? 500 : 0,
+            },
+          ];
```

Comment:
If there is no result from `prices` we should just return empty data then disable any UI